pkg/gatherers/conditional/gathering_rule.schema.json

```diff
@@ -120,6 +120,19 @@
                             "pattern": "^openshift-[a-zA-Z0-9_.-]{1,128}$"
                         }
                     }
+                },
+                "^api_request_counts$": {
+                    "type": "object",
+                    "title": "GatherApiRequestCountsParams",
+                    "required": [
+                        "alert_name"
+                    ],
+                    "properties": {
+                        "alert_name": {
+                            "type": "string",
+                            "title": "AlertName"
```

Comment:
I think it's better to validate it as well like in `alert_is_firing` condition where we have the pattern `^[a-zA-Z0-9_]{1,128}$`

Reply:
OK why not